pages/glossary/filename.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Filename
+key: filename
+---
+
+A filename is a text string that identifies an electronically stored file located at the end of a URL path excluding any query strings. For example the `src` attribute specifies a URL path of `src="/foo/bar.jpg?baz "` which contains the filename `bar.jpg`.
```

Comment:
I still think it would be good to differentiate between what a filename is, and where to find it in a URL. Perhaps:
"A filename is a text string that identifies an electronically stored file. In a URL it is located at the end of the path, after the last slash and before any query strings. For example ..." 
or something like that.

Reply:
I think it's a good idea. This will make the definition more clear.

Reply:
> I still think it would be good to differentiate between what a filename is, and where to find it in a URL. Perhaps:
> "A filename is a text string that identifies an electronically stored file. In a URL it is located at the end of the path, after the last slash and before any query strings. For example ..."
> or something like that.

Emma her suggestion solves the minor issue that this definition would only work in the context of this group. I would approve both implementations though.